### ISSUE TITLE
yara.so: undefined symbol: pe_get_directory_entry

### ISSUE THREAD

**Issue by wbenny** [VirusTotal/yara-python#36]
```
>>> import yara
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: /.../yara_python-3.5.0-py2.7-linux-x86_64.egg/yara.so: undefined symbol: pe_get_directory_entry
```

**Comment by plusvic** [comment]
Could you provide information about how you built yara and  yara-python?

**Comment by wbenny** [comment]
Exactly like you're saying in readme (except for using sudo for installation, because I'm using venv):
```
$ git clone --recursive https://github.com/VirusTotal/yara-python
$ cd yara-python
$ python setup.py build
$ python setup.py install
```

**Comment by wxsBSD** [comment]
This is b cause of an issue already open in yara-python repo. Can't find it right now, stuck on mobile.

**Comment by plusvic** [comment]
This should be solved by eba22f3eecf44788b42c4aea3eb8ed1b3b645892
